src/worldofzuul/CommandWords.java

```diff
@@ -7,31 +7,51 @@
  * @version 2006.03.30
  */
 
+
 public class CommandWords {
     private HashMap<String, CommandWord> validCommands;
-
+    
+    /**
+     * Creating an instance of CommandWords
+     * Creating an empty list 
+     * Validating if the commandword is known
+     * Storing if its valid
+     */
     public CommandWords() {
-        validCommands = new HashMap<String, CommandWord>();
-        for(CommandWord command : CommandWord.values()) {
-            if(command != CommandWord.UNKNOWN) {
-                validCommands.put(command.toString(), command);
+        validCommands = new HashMap<String, CommandWord>(); 
+        for(CommandWord command : CommandWord.values()) {   
+            if(command != CommandWord.UNKNOWN) {            
+                validCommands.put(command.toString(), command); 
             }
         }
     }
-
+    
+    /**
+     * Creating a getter method for CommandWord
+     * @param commandWord
+     * @return if command != null return command otherwise return unknown 
+     */
     public CommandWord getCommandWord(String commandWord) {
-        CommandWord command = validCommands.get(commandWord);
+        CommandWord command = validCommands.get(commandWord); 
```

Comment:
Det sidste space må gerne blive fjernet i linjen her.

Reply:
Fixed in 2f9347de89175dafc7d9307a43698c4a95ff66ce